app/models/user.rb

```diff
@@ -68,6 +69,10 @@ def active_ride
     Ride.where(driver_id: self.id).or(Ride.where(voter_id: self.id)).where(status: Ride.active_statuses).first
   end
 
+  def recent_complete_ride
+    self.rides.merge(Ride.completed).order('updated_at desc').first
```

Comment:
This is probably my misunderstanding, by why the merge, rather than just calling 'Ride.completed'?